### PR TITLE
School 397 добавление препроцессора scss

### PR DESCRIPTION
**Pull request by Rey-2903**

### CODE REVIEW

File: assets/styles/variables.scss [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    width: 100%;
+    background-color: $color-white;
+    border: 1px solid $color-black;
+    border-radius: 8px;
+}
+
+%title_container {
+    display: flex;
+    flex-direction: column;
+    margin: 0px 20px 16px 20px;
```
Comment (gaskeo) [inline assets/styles/variables.scss:59]: no need to write **px** when the value is **0**

File: assets/styles/variables.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    color: $color-blue;
+    user-select: none;
+    font-weight: 500;
+    font-size: 16px;
+}
+
+%closeButton {
+    cursor: pointer;
+    user-select: none;
+    margin: 8px 8px 8px 0px;
```
Comment (gaskeo) [inline assets/styles/variables.scss:74]: no need to write **px** when the value is **0** (`0px -> 0`)

File: assets/styles/variables.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+$color-green-active: #008B00;
+$color-white: #fff;
+$color-black: #000;
+$color-blue: #2698FF;
+$color-Error404: #737688;
+$color-gray: #808080;
+$color-title: #353747;
+
+%h1 {
+    font-family: 'Open Sans';
```
Comment (gaskeo) [inline assets/styles/variables.scss:11]: Add generic family name: `'Open Sans' -> 'Open Sans', sans-serif` (and in **h2** tag too). 
It's good practice: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family

File: components/components styles/EmployeeList.module.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+@import '../../assets/styles/variables.scss';
+
+.container {
+    @extend %components;
```
Comment (gaskeo) [inline components/components styles/EmployeeList.module.scss:4]: I can't find `%components` in **variables.scss**, maybe it's missed?

File: components/QueryComponents/QueryList.module.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
+@import '../../../assets/styles/variables.scss';
```
Comment (gaskeo) [inline components/QueryComponents/QueryList.module.scss:1]: `../../..` from this directory goes to directory, that doesn't contain **assets** folder. Maybe need `../..`?

**Comment by gaskeo** [comment]
CI build failed on `npm install yarn` command. On CI we use **node v16.x**, that can't be used with **node-sass v7.0.3**. You need to specify valid version of **node-sass** for our version of **node**. 

You can check all available versions of **node-sass** with this command: `npm view node-sass version`.

[Version compatibility of **node** and **node-sass**](https://www.npmjs.com/package/node-sass#Node-version-support-policy)

**Review by gaskeo — CHANGES_REQUESTED** [review]

File: assets/styles/variables.scss (hunk 2 above)
Comment (JIiCeHoK) [reply assets/styles/variables.scss:74]: Разницы нет. Можно и так оставить

**Review by gaskeo — APPROVED** [review]

**Pull request merged**